src/frontend/pages/chat/editor.js

```diff
@@ -0,0 +1,351 @@
+import { LitElement, html, css } from "lit";
+import { resolveMarkdown } from "lit-markdown";
+
+import "../../components/formatting-button";
+
+//da markdown a html
+export function parseMarkdown(text) {
```

Comment:
funzione da spostare in un apposito service

Reply:
risolto